src/components/visualizations/chart/chartOptionsBuilder.ts

```diff
@@ -716,7 +739,12 @@ export function generateTooltipFn(viewByAttribute: any, type: string, config: IC
         if (viewByAttribute) {
             // For some reason, highcharts ommit categories for pie charts with attribute. Use point.name instead.
             // use attribute name instead of attribute display form name
-            textData.unshift([customEscape(viewByAttribute.formOf.name), customEscape(point.category || point.name)]);
+            textData.unshift([
+                customEscape(viewByAttribute.formOf.name),
+                // since applying 'grouped-categories' plugin,
+                // 'category' type is replaced from string to object in highchart
+                customEscape(point.category.name || point.name)
```

Comment:
Is point.category always defined?

`point.category && point.category.name || point.name`